From: SeongJae Park <sj@kernel.org>
Date: Tue, 8 Oct 2024 17:45:50 -0700
Subject: [PATCH] ===== revert debugfs interface removal =====

Should continue testing and fixing DAMON debugfs interface until it is
totally removed.  For continuing the testing, revert the RFC changes of
the removal on damon/next tree.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/TGDVjBA4 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/TGDVjBA4

diff --git a/damon_meta_changes/TGDVjBA4 b/damon_meta_changes/TGDVjBA4
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

